inc/saiacl.h

```diff
@@ -1997,7 +2073,8 @@ typedef struct _sai_acl_api_t
     sai_remove_acl_range_fn             remove_acl_range;
     sai_set_acl_range_attribute_fn      set_acl_range_attribute;
     sai_get_acl_range_attribute_fn      get_acl_range_attribute;
-
+    sai_create_acl_table_group_fn       create_acl_table_group;
+    sai_remove_acl_table_group_fn       remove_acl_table_group;
```

Comment:
@AnbalaganNatchimuthu IMHO Scenario #1 and Scenarios #3 both should be supported by the ACL Model. I wouldn't recommend Scenario#2, however the ACL model does support that and as a community we should consider that scenario. 

From your examples of 2 rules bound to ports where i can see you really need port list, and you can use both #1 and #3. However, the value addition of the new acl model is when you scale the number of rules beyond ports when there are ACL tables you create for routing interfaces and you will realize port list is not sufficient when you have both port based routing interfaces and VLAN based router interfaces -- where Scenario#3 is not scalable for you. And Scenario #1 is really really useful to scale your application and handle the complexity in the SAI implementation.